README.md

```diff
@@ -1 +1,68 @@
-# Webpack-SetUp
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To Do List
+
+> In this project, you will build a simple HTML list of To Do tasks. The list will be styled according to the specifications listed later in this lesson. This simple web page will be built using webpack and served by a webpack dev server. 
+
+
+## Built With
+
+- Html
+- Css
+- Javascript Modules
+- Webpack
+
+## Live Demo (Not Available)
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Have a Desktop or Laptop Computer running on with Windows, Mac OS or Linux operating system.
+- Have git installed on the Computer
+- Have a browser install on the Computer
+
+### Setup
+
+- If it suits you, on your local computer, create a directory in your preferred location (e.g in the Desktop or Document directory)
+
+### Install
+
+- Open your Terminal/Powershell or any command line interface you use, then navigate to your created above directory or the directory you'll like to clone this project directory.
+- Type `https://github.com/NahnahAJ/To-Do-List.git` if your're using https with git or `git@github.com:NahnahAJ/To-Do-List.git` if your preferred is ssh and hit enter.
+- Navigate to the `To-Do List` directory on your local computer in your above chosen directory and open the `src/index.html` file.
```

Comment:
- [ ] I would suggest you please make sure that you write the proper step to run the project on the machine.
 For example, You need to do `npm install` then only you have to do `npm start` to start the project. please write those step